src/Controller/TestEmailController.php

```
@@ -0,0 +1,115 @@
+      $publishable_key = $config->get('publishable_key');
+      $site_uuid = $config->get('site_uuid');
+      $default_author = $config->get('default_author_email');
+
+      // Check all possible sources for secret key
+      $env_secret_key = $_ENV['BENTO_SECRET_KEY'] ?? getenv('BENTO_SECRET_KEY');
+      $state_secret_key = \Drupal::state()->get('bento_sdk.secret_key');
+      $config_secret_key = $config->get('secret_key');
+      
+      // Determine which secret key to use (same logic as BentoService)
+      $secret_key = null;
+      $secret_key_source = 'none';
```

PHPMD (2.15.0)

Avoid unused local variables such as '$secret_key_source'. (Unused Code Rules)

(UnusedLocalVariable)